lib/cc/engine/analyzers/command_line_runner.rb

```diff
@@ -15,10 +15,11 @@ def initialize(command, timeout = DEFAULT_TIMEOUT)
         def run(input)
           Timeout.timeout(timeout) do
             out, err, status = Open3.capture3(command, stdin_data: input)
-            if status.success?
+            if status && status.success?
               yield out
             else
-              raise ::CC::Engine::Analyzers::ParserError, "`#{command}` exited with code #{status.exitstatus}:\n#{err}"
+              exitstatus = status && status.exitstatus
+              raise ::CC::Engine::Analyzers::ParserError, "`#{command}` exited with code #{exitstatus}:\n#{err}"
```

Comment:
Should this be `exitstatus.inspect` so that we get the `nil` explicitly printed?